### DISCUSSION TITLE
Use GitHub Pages for Crowdfunding Project?

### DISCUSSION THREAD

**Discussion by GlifTek** [community/community#22775] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,<br>
This is my first time posting here.<br>
I’ve been using Github resources daily for about a year and have enjoyed all the great talent and help from the open source community here.</p>
<p>My question is this:</p>
<p>I am preparing a crowdfunding campaign for a multi-platform web-based collaborative engineering application.</p>
<p>The majority of the parts of this multi-faceted project will be open source, for everyone to explore and contribute to.</p>
<p>Also, a good amount of the funding will go to hiring talent from the open source community to refine its code base(s) and bring this project to an enterprise level application.</p>
<p>I’m wondering if the rules regarding the use of github pages allow for me to have them act as a main page for potential crowdfunding funders to come to and find information about the project.</p>
<p>Is this permitted, or are the use of github pages only specifically for open source and not anything with commercial potential?</p>
<p>Any info and advice regarding this would be very appreciated.</p>
<p>Thanks!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Welcome to the Community <a class="mention" href="/gliftek">@GlifTek</a></p>
<p>The content of Pages is at the user’s discretion, as long as it’s legal and doesn’t otherwise violate the Terms of Service. What you’re describing is absolutely fine, so please feel free to proceed.</p>
<p>Please let us know if there is anything else we can do for you and we will be happy to assist!</p>

**Comment by GlifTek** [comment]
<p>Thank you very much for your informative reply, and I will proceed!<br>
This is what I’ve been waiting to know.</p>
